src/Components/banner/banner.css

```diff
@@ -0,0 +1,53 @@
+.App {
+    display: flex;
```

Comment:
There are built-in css classes for this. Look at layout.css

Reply:
fix